src/renderer/dx/ScreenPixelShader.h

```diff
@@ -74,7 +78,7 @@ float4 main(float4 pos : SV_POSITION, float2 tex : TEXCOORD) : SV_TARGET
 
     // TODO:GH#3930 Make these configurable in some way.

```

Comment:
Well this TODO is going to be a lot easier now that you piped through the settings buffer.